Gemfile

```diff
@@ -27,10 +27,17 @@ gem 'bootsnap', '>= 1.1.0', require: false
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem 'rack-cors'
+gem 'active_model_serializers'
```

Comment:
@Mycobee I'm not familiar with this gem 'active_model_serializers'.
What are the pros/cons compared to something like fast_jsonapi?